source/develop/release-management/features/sla/hosted-engine-network-check.html.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Alternative Network Check for Self Hosted Engine
+category: feature
+authors: dholler
+feature_name: Alternative Network Check for Self Hosted Engine
+feature_modules: ovirt-hosted-engine-ha, cockpit-ovirt, ovirt-ansible-hosted-engine-setup, ovirt-hosted-engine-setup
+feature_status: Released
+---
+
+# Alternative Network Check for Self Hosted Engine
+
+## Summary
+
+Currently ovirt-hosted-engine-ha, the agent running on potential hosts to host
+the Self Hosted Engine VM, checks his network liveliness by pinging the
+gateway.
+This feature adds alternative ways to check network liveliness, to enable
+Self Hosted Engine in networks which does not allow pinging the gateway.
+
+The progress of the implementation and testing of this feature is tracked on
+[Bug 1659052 - [RFE] Consider alternative to ICMP ping for hosted engine network test](https://bugzilla.redhat.com/1659052)
+
+## Owner
+
+*   Name: [Dominik Holler](https://github.com/dominikholler)
+
+*   Email: <dholler@redhat.com>
+
+
+## Detailed Description
+
+Additional ways to check the network liveliness to be added are DNS, TCP and
```

Comment:
Both DNS and TCP need to be supported with IPv6.

Reply:
I added the two sentences "Like ping, the DNS and TCP check should work for IPv4 and IPv6." and "Even there is nothing special about IPv6 in this feature, it might be useful to
verify on IPv6."